python/scalellm/_C/__init__.pyi

```diff
@@ -74,6 +74,7 @@ class SequenceOutput:
 
 class RequestOutput:
     def __init__(self) -> None: ...
+    prompt: str
```

Comment:
add members here is not sufficient, this is just a interface definition. we need add a member in c++ part as well.